src/regfox/regfox_api.js

```diff
@@ -123,4 +125,41 @@ const getRegistrationInfoUrl = (id) => {
   return REGFOX_GET_REGISTRATION_URL.replace('${id}', id);
 };
 
-export { getRegistrationInfo, searchRegistrations, exchangeBearerToken, login, REGFOX_GRAPHQL_URL, REGFOX_EXCHANGE_TOKEN_URL, getRegistrationInfoUrl };
+/**
+ * Marks a registrant (technically a transaction of an order of a registered user) as completed (which means fully paid).
+ * See https://gist.github.com/raytingtw/8d0a083dc130c40818d71606e41b884e.
+ * Returns a promise Error if the network is down.
+ *
+ * All of these parameters can be pulled straight from the registrationInfo.
+ *
+ * @param {int} formId is the form (webpage) the user signed up with, could be hardcoded to 372652
+ * @param {int} registrationId of the registrant (**not** the Id), I think sometimes this is called the OrderId
+ * @param {int} transactionId of the registrant
```

Comment:
Silly fact: No int type in JS means this isn't valid jsdoc apparently! 

```suggestion
 * @param {number} formId is the form (webpage) the user signed up with, could be hardcoded to 372652
 * @param {number} registrationId of the registrant (**not** the Id), I think sometimes this is called the OrderId
 * @param {number} transactionId of the registrant
```

Reply:
Huh! Thanks for the heads up :D.